src/main/java/org/gridsuite/study/server/repository/DynamicSimulationParametersEntity.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.study.server.repository;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "dynamicSimulationParameters")
+public class DynamicSimulationParametersEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "startTime", columnDefinition = "numeric default 0")
+    private int startTime;
+
+    @Column(name = "stopTime", columnDefinition = "numeric default 0")
+    private int stopTime;
+
+    @Column(name = "solverId", columnDefinition = "varchar(2) default null")
```

Comment:
take more margin, varchar(2) but what if a new simulator comes and is longer? This is cheap we don't need to optimize so much

Reply:
Done, set varchar(255)